user-overrides/All-variables.css

```diff
@@ -1,3 +1,16 @@
+/* ==UserStyle==
+@name         WKED - USER Overrides
+@namespace    github.com/openstyles/stylus
+@version      1.0.0
+@license      MIT
+@description  Adds WKED styling to the ultimate timeline userscript
+@author       Sepitus
+@homepageURL  https://github.com/Sepitus-exe/WKElementaryDark
+@supportURL   https://github.com/Sepitus-exe/WKElementaryDark/issues
+==/UserStyle== */
+
+/* See https://community.wanikani.com/t/userscript-wanikani-ultimate-timeline/10516 */
```

Comment:
Just a thought about publishing.

Would it be wise to publish the override template as we plan on publishing the rest of the files. Any future changes to the published file will most likely wipe any user changes made to it and since stylus tends to update automatically by default, it could lead to bunch of issues down the line.

Subsequently, if we do not plan on publishing the override template, there is no real need for metadata.

Reply:
Hmm. I see your point. Definitely don't want users just editing anything that's published to userstyles.world (since those will be autoupdated). The instructions do say to *copy* the files, not edit in place, but I think you're probably right that it's better not to publish these to userstyles.world.

I'll make this clearer in the README and in the comments to the override styles (removing the UserStyle headers).

Reply:
I removed the UserStyle headers from the user override files (and also provided the link to the correct forum post!). Just publish the main stylesheet and user script stylesheets to `userstyles.world` when you are happy. Don't publish the user overrides -- we will just provide instructions for how users can create their own.

Reply:
I would actually like to have the overrides published and see them updated by their creators over time. Its just the template, that seemed dangerous, since its ment to be edited.

On that note, I would love for you to publish your own override yourself, to give you a total control over how, when and what happens with it. 😄